pkg/mei/mkhihdr.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mei
+
+// MKHIHdr is the header for every MKHI command, see
```

Comment:
do you expect commands for the ME to be implemented outside this package? if not, i'd unexport this

Reply:
I will unexport